wave-utils/src/main/resources/templates/conda/dockerfile-conda-file.txt

```diff
@@ -2,6 +2,12 @@ FROM {{base_image}}
 COPY --chown=$MAMBA_USER:$MAMBA_USER conda.yml /tmp/conda.yml
 RUN micromamba install -y -n base -f /tmp/conda.yml \
     {{base_packages}}
-    && micromamba clean -a -y
+    && micromamba env export --name base --explicit > environment.lock
+
+RUN echo ">>>>>>> CONDA_LOCK_START" \
+    && cat environment.lock \
+    && echo "<<<<<<< CONDA_LOCK_END"
+
+RUN micromamba clean -a -y
```

Comment:
it would be better to keep all in the same `RUN` otherwise it will create two more layers

Reply:
done